src/NzbDrone.Core/Indexers/Definitions/AnimeBytes.cs

```diff
@@ -56,6 +62,13 @@ protected override bool CheckIfLoginNeeded(HttpResponse httpResponse)
             return false;
         }
 
+        protected override Task<IndexerResponse> FetchIndexerResponse(IndexerRequest request)
+        {
+            var key = $"{request.HttpRequest.Url.FullUri}.{HashUtil.ComputeSha256Hash(Settings.ToJson())}";
+
+            return _searchCache.Get(key, () => base.FetchIndexerResponse(request), TimeSpan.FromMinutes(1));
```

Comment:
I think bumping to 5 minutes would work better here. One have to also note that `/scrape.php` bases results off search index and that is updated every 5 minutes, so new torrents will take up to 5 minutes to show up in results.

Reply:
It's very unlikely that you'd see 20 new groups in 5 minutes window on AB and even then you are limiting RSS scraping to 15 minutes (or so I heard), so I don't see how that would affect anything.

Not really set on changing it though; just thought out I point out that search index we use is not updated instantly and always incurs delay of up to 5m.